docs/how-to-guides/develop-a-flow/develop-standard-flow.md

```diff
@@ -310,4 +310,75 @@ Since `convert_to_dict` has a dictionary output with two keys: `category` and `e
 ![flow_output](../../media/how-to-guides/develop-standard-flow/flow_output.png)
 :::
 
-::::
\ No newline at end of file
+::::
+
+## Referencing external files/folders in a flow
```

Comment:
I think maybe put this as a separate page
![image](https://github.com/microsoft/promptflow/assets/47586720/6bc706c7-1d91-433c-9e43-ebf7e66ea008)

Reply:
And it will be easier to reference to in other places.

Reply:
@wangchao1230 Have added the doc with a new page, and have updated the refer link in quick-start.md.